install_scripts/templates/common/docker-install.sh

```diff
@@ -120,18 +120,17 @@ installDockerK8s() {
 }
 
 _installDocker() {
-    if [ "$LSB_DIST" = "rhel" ] || [ "$LSB_DIST" = "ol" ] || [ "$LSB_DIST" = "sles" ]; then
-        if [ -n "$NO_CE_ON_EE" ]; then
-            printf "${RED}Enterprise Linux distributions require Docker Enterprise Edition. Please install Docker before running this installation script.${NC}\n" 1>&2
-            exit 1
-        fi
+    _should_skip_docker_ee_install
```

Comment:
this is just a refactor, right, no logic changes?

Reply:
yeah